bigquery/datasets.rb

```diff
@@ -57,10 +54,9 @@ def delete_dataset project_id:, dataset_id:
   # project_id = "Your Google Cloud project ID"
   # dataset_id = "ID of the dataset to delete"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.